src/gluonnlp/data/__init__.py

```diff
@@ -39,10 +39,9 @@
 from .word_embedding_evaluation import *
 from .intent_slot import *
 
-
 __all__ = (['batchify'] + utils.__all__ + transforms.__all__ + sampler.__all__
            + dataset.__all__ + corpora.__all__ + sentiment.__all__
            + word_embedding_evaluation.__all__ + stream.__all__ + conll.__all__
            + translation.__all__ + registry.__all__ + question_answering.__all__
            + dataloader.__all__ + candidate_sampler.__all__ + intent_slot.__all__
-           + glue.__all__)  # pytype: disable=attribute-error
```

Comment:
No need to change this file?

Reply:
Usually there should be whitespace between the end of the code and the comment. So when you address above comments, please also re-add the whitespace in this line?